### ISSUE TITLE
Übersetzungen deutsch

### ISSUE THREAD

**Issue by Seqway** [ioBroker/ioBroker.flot#40]
Hallo,

wie auch den Screenshots des folgenden Beitrages zu sehen ist --> Bug in der Werteanzeige (Bar-Graph) #34

sind manche Worte oder Reiter noch in englischer Sprache wie die Reiter: Input Data, Presets, Options. in den Beschreibenden Texten der einzelnen Felder steht auch zB noch background usw.

Dies nur als Hinweis.

**Comment by GermanBluefox** [comment]
Ehrlich gesagt, würde ich mich über PR freuen.

**Comment by Seqway** [comment]
Was meinst du mit PR ?

Ich kann mich ja mal die Tage hinsetzen und mal markieren wo noch englisch drin ist und Übersetzen ? Meinst du das ?

**Comment by Hypnos3** [comment]
PR heißt Pull Request.

Du kannst im GitHub unter Code die Quellcode Dateien direkt editieren und einen PullRequest senden.

**Comment by Seqway** [comment]
Ah ok, das habe ich noch nie gemacht.
Aber einmal ist immer das erste Mal. In welchen Dateien muss ich denn stoebern und aendern?

**Comment by Seqway** [comment]
Muesste dann hier sein oder?

src/i18n/de/translations.json

Einfach aendern und neu hochladen?

**Comment by Hypnos3** [comment]
Ja, das müsste es sein. Wenn dort was fehlt kannst du das hier auch schreiben.

**Comment by Seqway** [comment]
Ich hab mal einen PR erstellt
